frontend/src/v5/ui/components/shared/creationInfo/creationInfo.styles.ts

```diff
@@ -21,11 +21,12 @@ export const CreationInfoContainer = styled.div`
 	display: flex;
 	${({ theme }) => theme.typography.caption};
 	color: ${({ theme }) => theme.palette.base.main};
-	padding-left: 10px;
 	gap: 2px;
 `;
 
-export const TruncateName = styled.div`
+export const CreationInfoValue = styled.span``;
+
+export const TruncateName = styled(CreationInfoValue)`
```

Comment:
If `TruncateName` is a styled `CreationInfoValue`, maybe the latter could be removed?